test/integration/bin/internal/run-integration-tests.sh

```diff
@@ -162,6 +162,18 @@ function create_image() {
     fi
 }
 
+function use_image() {
+    setup_gcloud
+    export TF_VAR_gcp_image="$IMAGE_NAME" # Override the default image name.
+    export SKIP_CONFIG=1                  # No need to configure the image, since already done when making the template
+}
+
+# deprecated
+#
+# Note 1: not sure what the goal of creating images was but it appears to be
+# misbehaving due to gcp not playing nicely with slashes in image names
+# Note 2: custom images are not using in any integration test path, not sure
+# what custom images were for but I'm guessing build caching?
```

Comment:
This code is originally from https://github.com/weaveworks/build-tools/tree/master/provisioning; it creates images with all the packages and images preloaded to save time in each test run.

Reply:
we found out the images are not actually used by the tests